dev/reloadable-server/src/main/scala/play/core/server/LagomReloadableDevServerStart.scala

```diff
@@ -83,6 +84,14 @@ object LagomReloadableDevServerStart {
           }
         }
 
+        val before = System.currentTimeMillis()
+        val address = InetAddress.getLocalHost
+        val after = System.currentTimeMillis()
+        if (after - before > 100) {
+          System.out.println(s"WARNING: Retrieving local host name ${address} took more than 100ms, this can create problems at startup with Lagom. \r\n" +
+            "You are probably using MACOS Sierra with a wrongly configured /etc/host . See https://thoeni.io/post/macos-sierra-java/ for a solution")
+        }
```

Comment:
Should we use "\n" rather than "\r\n" ?

Reply:
Makes sense, will resubmit squashing again

Reply:
Is there a logger available? If so, we would want to use that. If not, we should make one available. 😄

Reply:
Yeah, I was thinking the same

Reply:
Might as well use two separate `println` statements instead of an embedded newline.

Reply:
It would be a good idea to use ANSI colors to make it stand out, as well. See how `play.utils.Colors` is used elsewhere in the file. I think yellow or red would make sense.

Reply:
Will use play.utils.Colors adn two separate println, in RED